review_practice/react-app/src/components/Modal/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import * as Style from './style';
+
+export default ({ body, clickOutOfModalAreaHandle }) => {
```

Comment:
```suggestion
export default ({ children, clickOutOfModalAreaHandle }) => {
```
`children` 변수명을 사용하는게 조금 더 일반적입니다